docs/standard-library/rest/decorators.md

```diff
@@ -205,14 +205,23 @@ The `server` keyword can take a third parameter with parameters as necessary:
 
 #### `@useAuth`
 
-Specify the authentication for the service.
+Specify the authentication for the service with the `@useAuth` decorator on the service namespace.
+
+The decorator accepts a single security scheme, a tuple of security schemes (both are used),
+a union of security schemes (either can be used), or a union of tuples of security schemes.
+
+A simple example:
 
 ```cadl
 @service
 @useAuth(BasicAuth)
 namespace PetStore;
 ```
 
+See the [documentation in the Http library][authentication] for full details.
+
+[authentication]: https://github.com/microsoft/cadl/blob/main/docs/standard-library/rest/authentication.md
```

Comment:
```suggestion
[authentication]: https://microsoft.github.io/cadl/standard-library/rest/authentication
```

point to website instead?

Reply:
+1

Reply:
or do a relative url, docusaurus will automatically link it

Reply:
I think relative URLs are the right fix and I'll do that throughout.